src/js/player.js

```diff
@@ -3067,7 +3113,12 @@ class Player extends Component {
    *         A promise.
    */
   exitPictureInPicture() {
+    if (window.documentPictureInPicture && window.documentPictureInPicture.window) {
+      window.documentPictureInPicture.window.close();
```

Comment:
You may want to add a comment saying the `"unload"` event handler takes care of firing the `Player#leavepictureinpicture` event.

Reply:
Thanks for the feedback, added.

Reply:
Happy to help!